main/src/main/java/org/mobilitydata/gtfsvalidator/validator/AgencyConsistencyValidator.java

```diff
@@ -75,18 +74,26 @@ public void validate(NoticeContainer noticeContainer) {
                 commonTimezone.getId(),
                 agency.agencyTimezone().getId()));
       }
-      // FIXME: this should be a warning. See
-      // https://github.com/MobilityData/gtfs-validator/issues/549.
-      if (hasLanguage != agency.hasAgencyLang()
-          || (hasLanguage
-              && agency.hasAgencyLang()
-              && !commonLanguage.equals(agency.agencyLang()))) {
+    }
+
+    // agency_lang field is optional. All provided values must be the same for all agencies.
```

Comment:
From what I can tell the spec doesn't require all the agencies to have the same `agency_lang`? This isn't called out in the definition like `agency_timezone`.

If so this comment needs to be revised and the notice should be downgraded from ERROR to WARNING.

Reply:
This is a de-facto requirement for thousands of feeds at Google. I agree to change that to WARNING in this pull request but let's revisit that later.

A feed that provides different agency_lang is almost certainly making the wrong thing - we discussed that on https://github.com/MobilityData/gtfs-validator/pull/711 Having different agency_lang is ambiguous and there is no officially recommended way on how to treat that (is that a language for `agency_name`? for `route_long_name`? for `stop_name`? nobody knows!)

Reply:
I have introduced different types of notices: InconsistentAgencyLangNotice and InconsistentAgencyTimezoneNotice.

I agree that they have different severity and they should be treated differently.